contracts/interfaces/events/IBosonFundsEvents.sol

```diff
@@ -56,4 +56,5 @@ interface IBosonFundsBaseEvents {
         address payable mutualizerAddress,
         address executedBy
     );
+    event DRFeeReturnFailed(uint256 indexed exchangeId);
```

Comment:
```suggestion
    event DRFeeReturnFailed(
        uint256 indexed exchangeId,
        address indexed tokenAddress,
        uint256 returnAmount,
        address payable mutualizerAddress,
        address executedBy
    );
```
consistent with other events (DRFeeRequested/DRFeeReturned).
May be useful to update the subgraph (?)